modules/tinymce/src/themes/silver/main/ts/ui/sidebar/Sidebar.ts

```diff
@@ -103,8 +103,12 @@ const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig, showSi
     // Show the default sidebar
     if (Type.isString(showSidebar)) {
       Composing.getCurrent(slider).each((slotContainer) => {
-        SlotContainer.showSlot(slotContainer, showSidebar);
-        Sliding.immediateGrow(slider);
+        if (!SlotContainer.getSlot(slotContainer, showSidebar).isNone()) {
```

Comment:
This could potentially be simplified by just checking the name is in the panel configs instead. For example:
```
if (Type.isString(showSidebar) && Obj.has(panelConfigs, showSidebar)) {
```

Another thing I realised we missed here, is that the sidebar names will have been lowercased here. I mentioned this a while back but I forgot this fact as well.

- [ ] Ensure to lowercase the sidebar name so it matches the registry panelConfigs name